config/validate.go

```diff
@@ -99,6 +99,24 @@ func validateDatatype(k string, v any, typ string) string {
 		default:
 			return fmt.Sprintf("field %s must be a string, int, float, or bool", k)
 		}
+	case "sliceorscalar":
+		switch vt := v.(type) {
+		case string, int, int64, float64, bool:
+			// we're good
+		case []any:
+			// we need to check that the slice is all the same type
+			// if it's empty or 1 element, it's fine
+			if len(v.([]any)) > 1 {
```

Comment:
Should this be `len(v.([]any)) > 0` if  it's ok to only have 1 element?

Reply:
No, if it's only 1 element that means all elements are the same type. We only need to check once there's more than 1.